### ISSUE TITLE
expose jsdom instance for config in tests

### ISSUE THREAD

**Issue by zcrumbo** [rei/vunit#20]
This popped up while trying to write tests that involved `sessionStorage` - there's no way to do things like change jsdom's `window.location.href` without having access to the jsdom instance itself. 
Can we expose a reference to it as a global in setup script?

**Comment by peripateticus** [comment]
Definitely. One workaround that I've used is that you can set `window` properties via `global.window` that is set up via `jsdom-global`. I recently helped someone with this who was trying to write some tests around `window.WebSocket`.
